README.md

```diff
@@ -11,7 +11,7 @@ HLASM Language Support is an IDE extension that supports the High Level Assemble
 
 HLASM Language Support is part of the Che4z open-source project. Feel free to contribute at our [GitHub repository](https://github.com/eclipse-che4z/che-che4z-lsp-for-hlasm/). For project documentation, see the [wiki](https://github.com/eclipse-che4z/che-che4z-lsp-for-hlasm/wiki/). 
 
-HLASM Language Support is also part of [Code4z](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.code4z-extension-pack), an all-round package that offers a modern experience for mainframe application developers, including extensions for language support, data editing, testing, and source code management. For an interactive overview of Code4z, see the [Code4z Developer Cockpit](https://mainframe.broadcom.com/code4z-developer-cockpit).
+HLASM Language Support is also part of [Code4z](https://techdocs.broadcom.com/code4z), an all-round VS Code extension package that offers a modern experience for mainframe application developers, including tools for language support, data editing, testing, and source code management. For an interactive overview of Code4z, see the [Code4z Developer Cockpit](https://mainframe.broadcom.com/code4z-developer-cockpit).
```

Comment:
```suggestion
HLASM Language Support is also part of [Code4z](https://techdocs.broadcom.com/code4z), an all-round [VS Code extension package](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.code4z-extension-pack) that offers a modern experience for mainframe application developers, including tools for language support, data editing, testing, and source code management. For an interactive overview of Code4z, see the [Code4z Developer Cockpit](https://mainframe.broadcom.com/code4z-developer-cockpit).
```

Reply:
The Code4z readme will soon just point at the techdocs space for documentation. Me and Lukas agree people are unlikely to download it from the marketplace. So the marketplace link is not necessary, especially not with easteregg text.